clients/linux/src/app.rs

```diff
@@ -603,7 +603,17 @@ impl LbApp {
                     d.close();
                     let acctscr = &lb.gui.account;
                     acctscr.sidebar.tree.set_name(&id, &name);
-                    lb.messenger.send(Msg::RefreshSyncStatus);
+
+                    match lb.core.file_by_id(meta.id) {
+                        Ok(f) => {
+                            lb.gui.win.set_title(&f.name);
+                            lb.gui.account.set_search_field_text(&lb.core.full_path_for(&f));
```

Comment:
I made a variable for the account screen, so we can just use it here:
```suggestion
                            acctscr.set_search_field_text(&lb.core.full_path_for(&f));
```